PackageConfig.cmake.in

```diff
@@ -16,3 +16,4 @@ find_package(CycloneDDS REQUIRED)
 include("${CMAKE_CURRENT_LIST_DIR}/@PROJECT_NAME@Targets.cmake")
 
 check_required_components("@PROJECT_NAME@")
+include("${CMAKE_CURRENT_LIST_DIR}/idlcxx/Generate.cmake")
```

Comment:
Before `idlcxx` can be merged to `master`, `Generate.cmake` will be removed. Instead, the `Generate.cmake` file in the cyclonedds repository should gain a `LANGUAGE` keyword. The reason behind it being the real fix for [issue 697 in cyclonedds](https://github.com/eclipse-cyclonedds/cyclonedds/issues/697). The logic for scanning dependencies is probably not something we want to maintain in each repository.

Reply:
I'll leave this for now as what you are suggesting requires changes in cyclonedds and given that at this moment that the idlcxx branch is replaced by the xtypes branch and that one is not yet compatible with the idlcxx branch of cyclonedds-cxx i suggest to do these changes once the xtypes branch is merged to master.

Reply:
That'll work.